asserts/pool.go

```diff
@@ -351,9 +495,24 @@ func (p *Pool) ToResolve() (map[Grouping][]*AtRevision, map[Grouping][]*AtSequen
 				u.at.Revision = rev
 			}
 		}
-		u.exportTo(r, p.groupings)
+		u.exportTo(atr, p.groupings)
+	}
+
+	ats := make(map[Grouping][]*AtSequence)
+	for _, u := range p.unresolvedSequences {
+		seq := u.(*unresolvedSeqRec)
+		if seq.at.Revision == RevisionNotKnown {
+			rev, err := p.curSeqRevision(seq.at)
+			if err != nil {
+				return nil, nil, err
+			}
+			if rev != RevisionNotKnown {
+				seq.at.Revision = rev
```

Comment:
we never get here, which means the tests are missing some scenarios where we are actually updating something we have already or similar?

Reply:
Should be covered now.